projects/wonka-ui/karma.conf.js

```diff
@@ -21,7 +21,7 @@ module.exports = function (config) {
       reports: ['html', 'lcovonly', 'text-summary'],
       fixWebpackSourcePaths: true
     },
-    reporters: ['spec', 'kjhtml'],
+    reporters: ['progress', 'kjhtml'],
```

Comment:
We need "reporters: ['spec', 'kjhtml']" in order for karma-spec-reporter to work.

Reply:
My bad. Fixed.